examples/bri-3/test/sriUseCase.e2e-spec.ts

```diff
@@ -4,22 +4,29 @@ import { ethers } from 'ethers';
 import * as request from 'supertest';
 import { v4 } from 'uuid';
 import { AppModule } from '../src/app.module';
+import { keccak256 } from 'ethers/lib/utils';
+import * as circomlib from 'circomlibjs';
+import * as crypto from 'crypto';
 
 jest.setTimeout(120000);
 
 let accessToken: string;
 let app: INestApplication;
 let server: any;
 
-const supplierBPiSubjectPublicKey =
+const supplierBPiSubjectEcdsaPublicKey =
```

Comment:
@biscuitdey can we put all the constants for the e2e test in a constants.ts library file please

Reply:
Done.